universal-login-wallet/src/ui/createServices.ts

```diff
@@ -11,6 +11,7 @@ export interface Config {
   relayerUrl: string;
   jsonRpcUrl: string;
   tokens: string[];
+  rampApiKey?: string;
```

Comment:
Should be required here

Reply:
Are you sure? I think this config is used in tests where we don't have rampApiKey